From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Thu, 16 Apr 2020 08:43:04 +0200
Subject: [PATCH] net: mdio:gpio: fix typos in dev_dbg

Signed-off-by: Steffen Trumtrar <s.trumtrar@pengutronix.de>
---
 drivers/net/phy/mdio-gpio.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/phy/mdio-gpio.c b/drivers/net/phy/mdio-gpio.c
index affa31ae2c9b..80d2394f4b11 100644
--- a/drivers/net/phy/mdio-gpio.c
+++ b/drivers/net/phy/mdio-gpio.c
@@ -52,7 +52,7 @@ static struct mdio_gpio_info *mdio_gpio_of_get_info(struct device_d *dev)
 
 	ret = of_get_gpio_flags(dev->device_node, 0, &flags);
 	if (ret < 0) {
-		dev_dbg(dev, "failed to get MDC inforamtion from DT\n");
+		dev_dbg(dev, "failed to get MDC information from DT\n");
 		goto free_info;
 	}
 
@@ -61,7 +61,7 @@ static struct mdio_gpio_info *mdio_gpio_of_get_info(struct device_d *dev)
 
 	ret = of_get_gpio_flags(dev->device_node, 1, &flags);
 	if (ret < 0) {
-		dev_dbg(dev, "failed to get MDIO inforamtion from DT\n");
+		dev_dbg(dev, "failed to get MDIO information from DT\n");
 		goto free_info;
 	}
 
