From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Wed, 15 Jul 2020 11:27:49 +0200
Subject: [PATCH] Release 2020.07.0/customers/iotmaxx/gateway/20200715-2

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b31ba050c273..ff109c9811fd 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 2020
 PATCHLEVEL = 07
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20200715-2
 NAME = None
 
 # *DOCUMENTATION*
