From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Thu, 16 Apr 2020 15:06:14 +0200
Subject: [PATCH] Release 5.6/customers/iotmaxx/gateway/20200416-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 4d0711f54047..0126690abf4c 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 6
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20200416-1
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
