From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Tue, 2 Jun 2020 07:22:25 +0200
Subject: [PATCH] Release 5.7/customers/iotmaxx/gateway/20200602-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b668725a2a62..c60be83ceb61 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 7
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20200602-1
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
