From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Thu, 23 Jul 2020 08:25:40 +0200
Subject: [PATCH] Release 5.8/customers/iotmaxx/gateway/20200723-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index fe0164a654c7..a4133c00a86f 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 8
 SUBLEVEL = 0
-EXTRAVERSION = -rc4
+EXTRAVERSION = -rc4-20200723-1
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
