From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Wed, 15 Jul 2020 10:49:15 +0200
Subject: [PATCH] Release 5.8/customers/iotmaxx/gateway/20200715-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index fe0164a654c7..78460ab44653 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 8
 SUBLEVEL = 0
-EXTRAVERSION = -rc4
+EXTRAVERSION = -rc4-20200715-1
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
