--- /arch/arm/boot/dts/imx7d-iotmaxx-gw4100.dts	2020-08-04 12:26:44.000000000 +0200
+++ b/arch/arm/boot/dts/imx7d-iotmaxx-gw4100.dts	2020-08-04 13:31:13.620252222 +0200
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: GPL-2.0+
 /*
-* Copyright (C) 2020 Steffen Trumtrar <s.trumtrar@pengutronix.de>
+* Copyright (C) 2020 Steffen Trumtrar <s.trumtrar@pengutronix.de> 
+* Copyright (C) 2020 Niklas Sill <sill@iotmaxx.de> 
 */
 
 /dts-v1/;
@@ -303,7 +304,7 @@
 		reg = <0x20>;
 	};
 
-	eeprom: eeprom@50 {
+	eeprom1: eeprom@50 {
 		compatible = "atmel,24c64";
 		vcc-supply = <&reg_ldo_misc_3v3>;
 		pagesize = <32>;
@@ -321,6 +322,13 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c4>;
 	status = "okay";
+
+	eeprom2: eeprom@50 {
+		compatible = "atmel,24c64";
+		vcc-supply = <&reg_ldo_misc_3v3>;
+		pagesize = <32>;
+		reg = <0x50>;
+	};
 };
 
 &iomuxc {
