From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Fri, 9 Apr 2021 17:49:45 +0200
Subject: [PATCH] Release 5.8/customers/iotmaxx/gateway/20210409-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 24a4c1b97bb0..60621c9e0c73 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 8
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20210409-1
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
