From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Mon, 18 May 2020 12:33:01 +0200
Subject: [PATCH] Release 5.7/customers/iotmaxx/gateway/20200518-3

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 70def4907036..bda5292c5ad1 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 7
 SUBLEVEL = 0
-EXTRAVERSION = -rc1
+EXTRAVERSION = -rc1-20200518-3
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
