From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Wed, 13 May 2020 14:21:09 +0200
Subject: [PATCH] Release 5.7/customers/iotmaxx/boards/gateway/20200513-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 70def4907036..84a7c97b5586 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 7
 SUBLEVEL = 0
-EXTRAVERSION = -rc1
+EXTRAVERSION = -rc1-20200513-1
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
