From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Tue, 26 May 2020 14:56:59 +0200
Subject: [PATCH] Release 5.7/customers/iotmaxx/gateway/20200526-2

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 70def4907036..caf39e37ec3b 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 7
 SUBLEVEL = 0
-EXTRAVERSION = -rc1
+EXTRAVERSION = -rc1-20200526-2
 NAME = Kleptomaniac Octopus
 
 # *DOCUMENTATION*
