From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Thu, 27 May 2021 13:53:46 +0200
Subject: [PATCH] Release 5.12/customers/iotmaxx/gateway/20210527-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 78b0941f0de4..f63e90ba2283 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 12
 SUBLEVEL = 1
-EXTRAVERSION =
+EXTRAVERSION =-20210527-1
 NAME = Frozen Wasteland
 
 # *DOCUMENTATION*
