From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Thu, 27 May 2021 13:39:55 +0200
Subject: [PATCH] Release 2021.04.0/customers/iotmaxx/gateway/20210527-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 9a10bb5d0408..53e81f5c88e8 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 2021
 PATCHLEVEL = 04
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20210527-1
 NAME = None
 
 # *DOCUMENTATION*
